pipeline/tests/test_regressions.py

```diff
@@ -10,3 +13,159 @@ def test_issue_0002():
     node = build_fake_node(omcore.Person)
     data = node.to_jsonld()
     assert data["@type"] == "https://openminds.ebrains.eu/core/Person"
+
+
+def test_issue_0003():
```

Comment:
question: this is a very complicated test for testing this issue. What was there a reason for it?

A very simple case would be "alternateName" in Person (list of str, 1-N).

Reply:
I wanted to test something other than Person 😉

Reply:
Valid point :sweat_smile: